docs/UsingOmnisci/index.rst

```diff
@@ -1,27 +1,6 @@
 OmniSci
 =======
 
-This section describes usage related documents for the OmniSciDB-based engine of Modin.
```

Comment:
My first comment about this file is to rename it to `docs/UsingOmnisciOnNative/index.rst`.
Second, I am still thinking we should provide info on setting both omnisci storage format and native engine. Below we can say
```
There is a more simple way to enable ``OmniSci`` engine. You can only export `MODIN_STORAGE_FORMAT` environment variable and Modin itself defines the rest ones.
```

@Garra1980 , @vnlitvinov , @devin-petersohn , what do you think?

Reply:
@vnlitvinov , @devin-petersohn , what do you think?

Reply:
I'm siding with @Garra1980 here - there's no need to define both when it's just more verbose way of achieving the same result.

Reply:
We could put a section related only to setting omnisci storage format first and further provide more info on setting `IsExperimental`, `Engine` and `StorageFormat` to give users more understanding about the stuff and why/how it works.

Reply:
Yes, but that info about setting is not about Omnisci engine(or whatever it is called now), it looks as related to some general part of documentation so we can just put a link to that part in Omnisci section

Reply:
It should go to the docs on how `modin.config` works I think:, maybe even here: https://modin.readthedocs.io/en/latest/flow/modin/config.html#usage-guide

Reply:
@Garra1980 , not sure what you are saying is related to general part, info about setting respective env vars relates to this page.

@vnlitvinov, I don't quite get what you mean. Please, elaborate.

I proposed my suggestions. What do you think on those?

Reply:
I meant what @vnlitvinov said - we can put link to usage-guide and not mention explicitly all that env setting stuff on Omnisci page